proof.go

```diff
@@ -465,6 +465,204 @@ func (proof Proof) VerifyInclusion(h hash.Hash, nid namespace.ID, leavesWithoutN
 	return res
 }
 
+// VerifySubtreeRootInclusion verifies that a set of subtree roots is included in
+// an NMT.
+// Warning: This method is Celestia specific! Using it without verifying
+// the following assumptions, can return unexpected errors, false positive/negatives:
+// - The subtree roots are created according to the ADR-013
+// https://github.com/celestiaorg/celestia-app/blob/main/docs/architecture/adr-013-non-interactive-default-rules-for-zero-padding.md
+// - The tree's number of leaves is a power of two
+// The subtreeWidth is also defined in ADR-013.
```

Comment:
it's because this verification mechanism is Celestia specific. Implementing the generalised inner nodes inclusion verification is more involved and would require more time to do.